admin/main.go

```diff
@@ -0,0 +1,31 @@
+package main
+
+import (
+	"context"
+	"log/slog"
+	"net"
+	"os"
+	"time"
+
+	"github.com/Azure/ARO-HCP/admin/pkg/admin"
+)
+
+func main() {
+
+	logger := slog.New(slog.NewJSONHandler(os.Stdout, nil))
+
+	listener, _ := net.Listen("tcp", ":8446")
```

Comment:
```suggestion
	listener, err := net.Listen("tcp", ":8443")
	if err != nil {
	  panic(err)
	}
```
I know the port doesn't really "matter", but I think we should use 8080 or 8443 for consistency